src/router.jsx

```diff
@@ -2,11 +2,13 @@ import React from 'react'
 import { Router as ReachRouter } from '@reach/router'
 
 import App from '_views/app'
+import Main from '_views/main'
 import NotFoundPage from '_views/not-found'
 
 const Router = () => (
   <ReachRouter>
-    <App path="/" />
+    <App path="/login" />
+    <Main path="/users" />
```

Comment:
Don't call your login App
```suggestion
    <Login path="/login" />
    <App path="/users" />
```